gatsby-browser.js

```diff
@@ -22,3 +22,7 @@ export const onClientEntry = () => {
 export const shouldUpdateScroll = () => {
   return false
 }
+
+export const onRouteUpdate = () => {
```

Comment:
I think this will need to sit on the menu button itself 
Main/top menu and footer links

```
dataLayer.push({
	'event':'e_menuNav',
	'menuName':'[MENU NAME]',
	'menuType':'[MENU TYPE]'
});
```
Dynamic parameters:
[MENU NAME]: e.g. “pricing”, “blog”, “case studies”
[MENU TYPE]: e.g. “header”, footer”

probably add it to onClick on the items
```javascript
const handleGTMClick = (menuName: string) => {
 window.dataLayer.push({
	event: 'e_menuNav',
	menuType: 'header' // or 'footer'
	menuName,
  })
}
```

Reply:
Cool cool. updated accordingly